docs/_snippets/build-storybook-production-mode.md

```diff
@@ -6,14 +6,14 @@ ng run my-project:build-storybook
 ```
 
 ```shell renderer="common" language="js" packageManager="npm"
-npm run build-storybook
+npm run storybook build
```

Comment:
@ahayes91, can you revert this back? It was correct because it referred to the build script automatically added when Storybook is added to the user's project.